packages/utils/upgrade/src/cli/index.ts

```diff
@@ -1,32 +1,62 @@
 import os from 'os';
 import chalk from 'chalk';
-import { program } from 'commander';
+import { Option, program } from 'commander';
 
 import { version as packageJSONVersion } from '../../package.json';
 import { Version } from '../modules/version';
 
 import type { CLIOptions } from './types';
 
+const projectPathOption = new Option(
+  '-p, --project-path <project-path>',
+  'Path to the Strapi project'
+);
+
+const dryOption = new Option(
+  '-n, --dry',
```

Comment:
Reminder for later: we should probably make it --dry-run instead of --dry (or include both) since dry-run is more common